src/core/Akka/Actor/ActorRef.cs

```diff
@@ -110,9 +127,6 @@ protected override void TellInternal(object message, IActorRef sender)
             
             switch (message)
             {
-                case ISystemMessage msg:
```

Comment:
As mentioned in #7501, this code never worked because it was in the wrong method. Removing it should actually speed up `Ask<T>` processing slightly.